src/panel/poi_panel.js

```diff
@@ -85,6 +85,17 @@ PoiPanel.prototype.setPoi = async function (poi) {
   await this.panel.update()
 }
 
+PoiPanel.prototype.copy = function () {
+  let copyUrl = document.getElementById('share-url')
+  copyUrl.select()
+  document.execCommand('copy')
+  copyUrl.blur()
+}
+
+PoiPanel.prototype.center = function() {
+  fire('fit_map', this.poi, {sidePanelOffset : true})
```

Comment:
This does not respect the `ZOOM_BY_POI_TYPES` value.  
This is due to how the poi is initialized in `Poi.apiLoad`. The API does not currently return a `type` and the default zoom value is used. 
I think the type `"poi"` should be set by default when loading a Poi from the API.

Reply:
fixed